test/scripts/hello_world_on_chain.txtar

```diff
@@ -0,0 +1,5 @@
+# [execute]
+exec npx hardhat test --grep $TEST_CASE
```

Comment:
does `pkgs.nodejs_18` cover all the node dependencies? I haven't run any of the node examples so I wasn't sure.

Reply:
I would suggest against `node_18`.  It think it will work, but it's end of life was April 25, 2025 and so it is starting to be removed from many places.  I think I was using `node_22` when working on the sol based TAParser, but I don't know what version @mwittie was using.

Reply:
My system was 18, but I've updated to 22 and tested. Also bumped the nix shell version to 22 and that works as well